plugins/gui/src/LogConsole.cpp

```diff
@@ -197,18 +197,17 @@ void megamol::gui::LogConsole::Update(WindowCollection::WindowConfiguration& wc)
             }
 
             // Check for screenshot privacy note
-            auto note_pos = entry.message.find("<<<< PRIVACY NOTE >>>>>");
+            auto note_pos = entry.message.find("[Screenshot]");
             if (note_pos != std::string::npos) {
                 this->screenshot_note_show = true;
-                this->screenshot_note = entry.message.substr(note_pos);
+                this->screenshot_note = entry.message;
```

Comment:
Maybe instead of checking for screenshot warnings every time we can make the screenshot note data driven, e.g. the screenshot UI note is triggered when a screenshot is taken by the service?